question-service/index.js

```diff
@@ -0,0 +1,35 @@
+import bodyParser from "body-parser";
+import dotenv from "dotenv";
+import express from "express";
+import mongoose from "mongoose";
+import corsMiddleware from "./middlewares/cors.js";
+import errorHandler from "./middlewares/errorHandler.js";
+import loggingMiddleware from "./middlewares/logging.js";
+import router from "./router/router.js";
+
+
+dotenv.config();
+const app = express();
+const port = process.env.DEV_PORT || 8003;
+
+
+app.use(corsMiddleware);
+app.use(loggingMiddleware);
+app.use(bodyParser.json());
```

Comment:
i think you might also need cookieParser middleware to parse cookies from HTTP requests